phpunit.xml.dist

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    backupGlobals="false"
+    backupStaticAttributes="false"
+    colors="true"
+    convertErrorsToExceptions="true"
+    convertNoticesToExceptions="true"
+    convertWarningsToExceptions="true"
+    processIsolation="false"
+    stopOnFailure="false"
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd"
+>
+    <coverage>
+        <include>
+            <directory suffix=".php">./src/</directory>
+        </include>
+    </coverage>
+    <testsuites>
+        <testsuite name="Storm Library Test Suite">
+            <directory>./tests</directory>
+        </testsuite>
+    </testsuites>
+</phpunit>
```

Comment:
Why was this added?

Reply:
It's a testing config for PHPUnit. It was added so instead of running `./vendor/bin/phpunit ./tests`, we could just run `./vendor/bin/phpunit` instead, and also allows us to use a code coverage service in the future, if we so choose.

We have a similar config set up in the main repo as well (https://github.com/wintercms/winter/blob/develop/phpunit.xml), but this one allows us to run the library tests in isolation.

Reply:
What's with the .dist extension?

Reply:
`phpunit.xml.dist` is standard for reusable libraries (which Storm arguably is). It's a distributed "example" config which can be overridden locally by a `phpunit.xml`, but will be used if `phpunit.xml` doesn't exist.